yt/yt/server/job_proxy/stderr_writer.h

```diff
@@ -48,16 +47,17 @@ class TStderrWriter
         NConcurrency::IThroughputThrottlerPtr throttler);
 
     size_t GetCurrentSize() const;
-    TString GetCurrentData() const;
+    NApi::TPagedLog GetCurrentData(const NApi::TPagedLogReq& request) const;
 
 private:
     void DoWrite(const void* buf, size_t len) override;
 
-    void SaveCurrentDataTo(IOutputStream* output) const;
+    void SaveCurrentDataTo(IOutputStream* output, bool continuous = false) const;
```

Comment:
Honestly, I don't like `continuous` here. Maybe `addPrefix` or something like this?